API/PROPERTIES

```diff
@@ -33,6 +33,11 @@
     Until we find the answer to those questions (for each and every alias)
     we will continue to regard aliases as noise and thus remove/deprecate them.
 
+    CONSEQUENCE
+
+    "Constr" module does not appear in the API because whatever it provides
+    is also provided by "Term" (which is already present in the API).
+
```

Comment:
This is in conflict with PR #808 which (among many good ideas) also unfortunately repopulates API with many aliases.